src/BlockTypesController.php

```diff
@@ -228,6 +228,7 @@ protected function get_block_types() {
 		if ( Package::feature()->is_experimental_build() ) {
 			$block_types[] = 'ProductCollection';
 			$block_types[] = 'ProductRatingStars';
+			$block_types[] = 'ProductRatingCounter';
```

Comment:
Totally nitpicking here, so forgive me, but we could order them alphabetically 😅 
```suggestion
			$block_types[] = 'ProductRatingCounter';
			$block_types[] = 'ProductRatingStars';
```